examples/lua-hello/Makefile

```
@@ -3,6 +3,10 @@
 # Specify this directory relative to the current application.
 TOCK_USERLAND_BASE_DIR = ../..
 
+# We only compile this for Cortex-M platforms because of compiler
+# errors when building lua on risc-v.
+TOCK_TARGETS = cortex-m0 cortex-m3 cortex-m4
```

```suggestion
TOCK_TARGETS := cortex-m0 cortex-m3 cortex-m4
```

Generally avoiding lazy evaluation makes all of the rest of the machinery more reliable.